### PR TITLE
Ship both x86 and aarch64 darwin libs

### PR DESCRIPTION
**Pull request by atoulme**
Signed-off-by: Antoine Toulme <antoine@lunar-ocean.com>

### CODE REVIEW

File: ipa-multipoint/src/test/java/org/hyperledger/besu/nativelib/ipa_multipoint/LibIpaMultipointTest.java [hunk 1]
```diff
@@ -17,36 +17,14 @@
...
-    } else if (macFile.exists()) {
-      System.load(macFile.getAbsolutePath());
-    } else {
-      throw new RuntimeException("could not setup jni path for test");
-    }
-  }
-
   @Test
-  public void testCallLibrary() {
+  public void testCallLibrary() throws Exception {
```
Comment (shemnon) [inline ipa-multipoint/src/test/java/org/hyperledger/besu/nativelib/ipa_multipoint/LibIpaMultipointTest.java:27]: ?? Why add the generic `throws Exception` ?? None of the called apis were changed.

**Review by shemnon — APPROVED** [review]
Looks good for the most part.

Except that the Blake2F lib performs worse than the Java lib right now on mac. Performance is out of scope for this PR.

**Review by garyschulte — COMMENTED** [review]
Are we not able to build a lipo/multi-arch darwin lib for blake2f in the same way we do for the other besu-native libs?

File: ipa-multipoint/src/test/java/org/hyperledger/besu/nativelib/ipa_multipoint/LibIpaMultipointTest.java (hunk 1 above)
Comment (atoulme) [reply ipa-multipoint/src/test/java/org/hyperledger/besu/nativelib/ipa_multipoint/LibIpaMultipointTest.java:27]: Probably an artifact from the back and forth during testing. I can remove that change, sure.

**Comment by atoulme** [comment]
jna now uses separate folders for darwin, so you can't use a multi-arch lib.
And blake2bf has some ever so slightly differences in imports between m1 and arm, and required a change in the code for imports. You can probably reconcile and use a precompile condition like `ifdef __APPLE__`. I picked that up a bit late.

**Comment by atoulme** [comment]
I have verified locally that the m1 lib for ipa-multipoint now loads properly with the zulu java 17 SDK.

**Pull request merged**